### DISCUSSION TITLE
一分钟读论文：《通过 JS 分类即时加速移动网页》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#13] [Announcements]
# 一分钟读论文：《通过 JS 分类即时加速移动网页》

通过机器学习(ML)加持的 JS 分类服务优化移动页面加载时间减少了 50%，用户体验提升 60% 以上，同时保持了大多数页面 90%-100% 的交互体验。 诺基亚⻉尔实验室和美国纽约大学、美国兰卡斯特大学管理学院合著的论文《To Block or Not to Block: Accelerating Mobile Web Pages On-The-Fly Through JavaScript Classification》 设计了一个服务 - SlimWeb 通过识别并删除网页的⾮关键 JS 以优化性能。

https://unbug.github.io/To-Block-or-Not-to-Block-Accelerating-Mobile-Web-Pages-On-The-Fly-Through-JavaScript-Classification/

**Comment by unbug** [comment]
用户体验提高60%以上，这个产品经理该同意搞了吧
